lib/cryptoexchange/exchanges/novaexchange/models/market_pair.rb

```diff
@@ -0,0 +1,14 @@
+module Cryptoexchange::Exchanges
+  module Novaexchange
+    module Models
+      class MarketPair < Cryptoexchange::Models::MarketPair
+        attr_accessor :name
+
+        def initialize(params={})
+          super
+          @name = params[:name]
```

Comment:
ESP/DOGE - its name is ESP_BLRY. @tmlee thinks it's likely a bug on their side, which I agree. However, querying ESP_DOGE gives a 404.